File: bookBeauty.Services/Database/RecommendResult.cs
using System;
using System.Collections.Generic;

namespace bookBeauty.Services.Database;

public partial class RecommendResult
{
    public int RecommendResultId { get; set; }

    public int? ProductId { get; set; }

    public int? FirstProductId { get; set; }

    public int? SecondProductId { get; set; }

    public int? ThirdProductId { get; set; }
}

File: UI/bookbeauty_desktop/lib/models/recommend_result.dart
import 'package:json_annotation/json_annotation.dart';

part 'recommend_result.g.dart';

@JsonSerializable()
class RecommendResult {
  int? recommendResultId;
  int? firstproductId;
  int? secondproductId;
  int? thirdproductId;

  RecommendResult(
      {this.recommendResultId,
      this.firstproductId,
      this.secondproductId,
      this.thirdproductId});

  factory RecommendResult.fromJson(Map<String, dynamic> json) =>
      _$RecommendResultFromJson(json);

  Map<String, dynamic> toJson() => _$RecommendResultToJson(this);
}
